eng/pipelines/templates/jobs/ci.yml

```diff
@@ -102,7 +128,7 @@ jobs:
     timeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
 
     dependsOn:
-      - 'Build'
+      - 'Build_Extended'
```

Comment:
Any reason Analyze cannot run after Build? Are there artifacts being used from the Build_Extended in the Analyze job?